apps/website/src/components/games/GameCard.tsx

```diff
@@ -12,14 +12,20 @@ interface Props {
 function GameCard({ name, tags, linkHref, imgSrc }: Props) {
   const router = useRouter()
 
-  return (
-    <Card
-      name={name}
-      tags={tags}
-      imgSrc={imgSrc}
-      onClick={linkHref ? () => router.push(linkHref) : undefined}
-    />
-  )
+  const addToRouter = () => {
+    const isExternal = linkHref.startsWith('http') || linkHref.startsWith('www')
```

Comment:
I would not do this check again here. instead, just do not pass linkHref to GameCard if it is an external link (as then the a is around it and will handle the link). basically, onClick then checks only for the presence of linkHref, and if it is there, links to it, otherwise does nothing

Reply:
very true, thanks! I tried something similar but it didn't work as the card (button) was disabled, but now it does. Def. cleaner this way!